multiversx_sdk_cli/contract_verification.py

```diff
@@ -80,6 +77,38 @@ def trigger_contract_verification(
     contract_verification = ContractVerificationRequest(contract, source_code, signature, docker_image)
 
     request_dictionary = contract_verification.to_dictionary()
-    response = requests.post(verifier_url, json=request_dictionary)
 
-    return response
+    url = f"{verifier_url}/verifier"
+    response = requests.post(url, json=request_dictionary).json()
+
+    status = response.get("status", "")
+    if status:
+        logger.info(f"Task status: {status}")
+
+        if status == "error":
+            dump_out_json(response)
+        else:
+            dump_out_json(response)
+    else:
+        task_id = response.get("taskId", "")
```

Comment:
an error can be also of other causes, so taskId should be checked here, and if it does not exist, dump the error in a standard way

Reply:
will do!